### PR TITLE
[macOS] week 47 software update

### PR DESCRIPTION
**Pull request by mikhailkoliada**
# Description

Updates:

 Xamarin.iOS     : 15.2.0.17
 Xamarin.Mac     : 8.2.0.17

## Check list
- [X] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/macos/toolsets/toolset-10.15.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
             "6.12.0.140", "6.10.0.106", "6.8.0.123", "6.6.0.166", "6.4.0.208"
         ],
         "ios-versions": [
-            "14.14.2.5", "14.10.0.4", "14.8.0.3", "14.6.0.15", "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2", "13.18.2.1", "13.16.0.13", "13.14.1.39", "13.10.0.21", "13.8.3.0", "13.6.0.12", "13.4.0.2", "13.2.0.47"
+            "15.2.0.17", "14.14.2.5", "14.10.0.4", "14.8.0.3", "14.6.0.15", "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2", "13.18.2.1", "13.16.0.13", "13.14.1.39", "13.10.0.21", "13.8.3.0", "13.6.0.12", "13.4.0.2", "13.2.0.47"
```
Comment (al-cheb) [inline images/macos/toolsets/toolset-10.15.json:26]: Xamarin.IOS&Mac doesn't support Catalina.

File: images/macos/toolsets/toolset-11.json [hunk 2]
```diff
@@ -16,10 +16,10 @@
             "6.12.0.140"
         ],
         "ios-versions": [
-            "15.2.0.1", "15.0.0.6", "14.20.0.24", "14.16.0.5", "14.14.2.5", "14.10.0.4", "14.8.0.3", "14.6.0.15", "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2"
+            "15.2.0.17", "15.2.0.1", "15.0.0.6", "14.20.0.24", "14.16.0.5", "14.14.2.5", "14.10.0.4", "14.8.0.3", "14.6.0.15", "14.4.1.3", "14.2.0.12", "14.0.0.0", "13.20.2.2"
```
Comment (al-cheb) [inline images/macos/toolsets/toolset-11.json:19]: We should update 15.2.0.1 -> 15.2.0.17

File: images/macos/toolsets/toolset-11.json [hunk 3]
```diff
@@ -16,10 +16,10 @@
...
         ],
         "mac-versions": [
-            "8.2.0.1", "7.14.0.27", "7.10.0.5", "7.8.2.5", "7.4.0.10", "7.2.0.3", "7.0.0.15", "6.22.1.26", "6.20.2.2"
+            "8.2.0.17", "8.2.0.1", "7.14.0.27", "7.10.0.5", "7.8.2.5", "7.4.0.10", "7.2.0.3", "7.0.0.15", "6.22.1.26", "6.20.2.2"
```
Comment (al-cheb) [inline images/macos/toolsets/toolset-11.json:22]: We should update 8.2.0.1 -> 8.2.0.17

File: images/macos/toolsets/toolset-12.json [hunk 4]
```diff
@@ -12,10 +12,10 @@
             "6.12.0.140"
         ],
         "ios-versions": [
-            "15.2.0.1", "15.0.0.6"
+            "15.2.0.17", "15.2.0.1", "15.0.0.6"
```
Comment (al-cheb) [inline images/macos/toolsets/toolset-12.json:15]: It should be the same as Big Sur.

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**